src/components/courseCard/index.js

```diff
@@ -3,15 +3,16 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
 import StarRatings from 'react-star-ratings';
+import { withTranslation } from 'react-i18next';
 
 import Img from '../img/index';
 import { links } from '../../utils/constants';
 
 import styles from './style.module.scss';
 
-export const CourseCard = course => {
-  const { title, authors, rate, price = 'For free', img = '', id } = course;
-
+const CourseCard = props => {
+  const { title, authors, rate, price = 'For free', img = '', id, t } = props;
+ 
```

Comment:
what does it mean 't'? Is it possible to replace this variable with 'localization' or 'translator' naming?

Reply:
This is a function from i18 library, the custom name is 't'. Do you wanna rename it to 'translator'?

Reply:
Yep, of course, if it is possible.) On my mind, It is more readable.